src/vs/platform/theme/common/themeService.ts

```diff
@@ -20,6 +20,7 @@ export const HIGH_CONTRAST = 'hc';
 export type ThemeType = 'light' | 'dark' | 'hc';
 
 export interface ITheme {
+	readonly name: string;
```

Comment:
The reason for adding a new field here is give us a human readable identifier to workwith in css rules. The `selector` is something like `vs.vscode-theme-solarized-light-themes-solarized-light-color-theme-json` for example. 

I wasn't exactly sure what this new field should be called though. `name` seemed like the best fit but `label` and `id`are also used in the codebase